roamance/src/api/errors.ts

```
@@ -0,0 +1,9 @@
+export class ApiError extends Error {
+  status: number;
+
+  constructor(message: string, status: number = 500) {
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)